### PR TITLE
Tribulation fix + effects to discord masteries

### PR DESCRIPTION
**Pull request by kevin-10**

### CODE REVIEW

**Comment by kevin-10** [comment]
Curses Fixes:
- Added tables to track curse, feeblemind, clumsy and weaken.  With EA changing the many of the curse removals are done, they all have to be identified.  Since Curse and the 3 weaker curses share mod names, this was impossible before.

Added clilocs to rock hammer, and removed unneeded cliloc from PlumTreeAddon.

File: Scripts/Spells/Skill Masteries/BardSpells/Tribulation.cs [hunk 1]
```diff
@@ -140,8 +140,9 @@ public override void DoDamage(Mobile victim, int damageTaken)
 				
 				double reduce = DamageModifier(victim);
 				
-				damage -= (int)((damage * reduce) * m_SlayerBonus);
```
Comment (Argalep) [inline Scripts/Spells/Skill Masteries/BardSpells/Tribulation.cs]: Errors:
 + Spells/Skill Masteries/BardSpells/Tribulation.cs:
    CS0266: Line 144: Cannot implicitly convert type 'double' to 'int'. An expli
cit conversion exists (are you missing a cast?)

**Comment by kevin-10** [comment]
Weird, the travis thing passed, and I made sure it compiled.

**Comment by kevin-10** [comment]
Check it now.

**Pull request merged**